sakura_core/dlg/CDlgPluginOption.cpp

```diff
@@ -139,17 +139,16 @@ void CDlgPluginOption::SetData( void )
 		wstring sSection;
 		wstring sKey;
 		wstring sValue;
-		wstring sType;
 
 		cOpt->GetKey(&sSection, &sKey);
 		if( sSection.empty() || sKey.empty() ) {
-			sValue = L"";
+			sValue.clear();
 		}
 		else {
 			if( !cProfile->IOProfileData( sSection.c_str(), sKey.c_str(), sValue ) ){
 				// Optionが見つからなかったらDefault値を設定
 				sValue = cOpt->GetDefaultVal();
-				if( sValue != wstring(L"") ){
+				if( sValue != wstring() ){
```

Comment:
```suggestion
				if( sValue.length() ){
```

Reply:
@berryzplus fix it https://github.com/sakura-editor/sakura/pull/1860/commits/1fc1801a340c0d1ebb2b306d27b3d48c14ec2c17